src/providers/store-provider/storeContext.ts

```diff
@@ -0,0 +1,16 @@
+import { createContext } from 'react';
+
+export interface IStoreContext {
+  selectedCity: string;
+}
+
+export type StoreDispatchType = (storeProperty: Partial<IStoreContext>) => void;
```

Comment:
This is not used outside, please remove `export`.
The same for `defaultStore` below